wnfs/src/private/file.rs

```diff
@@ -1307,7 +1366,10 @@ mod proptests {
                 .await
                 .unwrap();
             disk_file.read_exact(&mut source_content).await.unwrap();
-            let wnfs_content = file.read_at(offset, size, forest, store).await.unwrap();
+            let wnfs_content = file
+                .read_at(offset as u64, Some(size), forest, store)
```

Comment:
`offset` should probably just be generated as `u64`. So in the argument of the function:
`#[strategy(0..FIXTURE_SCHERZO_SIZE as u64)] offset: u64,`

Reply:
Done 👍